README.md

```diff
@@ -41,6 +57,39 @@ functions and are stored on a single central place, in a relational database. Co
 checkpoints is not only helpful for complying with strict regulatory frameworks, but also helps during development 
 and debugging of your Spark-based data processing.
 
+## Motivation
+
+Big Data strategy for a company usually includes data gathering and ingestion processes.
+That is the definition of how data from different systems operating inside a company
+are gathered and stored for further analysis and reporting. An ingestion processes can involve
+various transformations like:
+* Converting between data formats (XML, CSV, etc.)
+* Data type casting, for example converting XML strings to numeric values
+* Joining reference tables. For example this can include enriching existing
+  data with additional information available through dictionary mappings.
+  This constitutes a common ETL (Extract, Transform and Load) process.
+
+During such transformations, sometimes data can get corrupted (e.g. during casting), records can
+get added or lost. For instance, *outer joining* a table holding duplicate keys can result in records explosion.
+And *inner joining* a table which has no matching keys for some records will result in loss of records.
+
+In regulated industries it is crucial to ensure data integrity and accuracy. For instance, in the banking industry
+the BCBS set of regulations requires analysis and reporting to be based on data accuracy and integrity principles.
+Thus it is critical at the ingestion stage to preserve the accuracy and integrity of the data gathered from a
+source system.
+
+The purpose of Atum is to provide means of ensuring no critical fields have been modified during
+the processing and no records are added or lost. To do this the library provides an ability
+to calculate *hash sums* of explicitly specified columns. We call the set of hash sums at a given time
```

Comment:
do we want to explicitly word it like this, `hash sums`? I think if we are talking about a general measurement perhaps `hash sum` is a bit too specific / a bit misleading?

Reply:
very good text otherwise!

Reply:
Henestly I didn't write it. I just copied it from _Atum_ and adjusted the function name. It's because last time Manny asked about the supported functions and I found out we don't have them, but old Atum has.

Reply:
oh okay. No worries. But perhaps still, consider changing it